development.md

```diff
@@ -31,14 +31,16 @@ This is essentially equivalent to running `make image-push TAG=<tag>` in the exa
 1. Create a PR for your changes. Once merged, it will trigger a workflow to build and push the images for all the examples, 
 with the tag `stable`. This consistent tag name is used so that the tags in the [E2E test pipelines](https://github.com/numaproj/numaflow/tree/main/test) do not need to be 
 updated each time a new change is made. 
-2. After the changes have been merged it is encouraged to update the dependency management files so that the version 
+2. If the change that you have introduced is a breaking change, i.e. adds new features/logic, then after merge it is encouraged to update the dependency management files so that the version 
```

Comment:
```suggestion
2. If the change that you have introduced is a breaking change, then after merge you should update the dependency management files so that the version 
```